docs/products/caching/concepts/lua-scripts-caching.md

```diff
@@ -1,15 +1,14 @@
 ---
-title: Lua scripts with Aiven for Redis®*
+title: Lua scripts with Aiven for Caching
 ---
 
-Learn how to leverage the inbuilt support for Lua scripting in Redis®\*
-using Aiven.
+Learn how to leverage the inbuilt support for Lua scripting in Redis®* using Aiven for Caching.
```

Comment:
we probably wanna chagen this sentence, something like in Aiven for Caching?

Reply:
Done